docs/src/main/sphinx/installation/deployment.rst

```diff
@@ -143,10 +143,12 @@ The following provides a good starting point for creating ``etc/jvm.config``:
     -Djdk.nio.maxCachedBufferSize=2000000
     -XX:+UnlockDiagnosticVMOptions
     -XX:+UseAESCTRIntrinsics
+    -XX:-G1UsePreventiveGC
 
 Because an ``OutOfMemoryError`` typically leaves the JVM in an
 inconsistent state, we write a heap dump, for debugging, and forcibly
-terminate the process when this occurs.
+terminate the process when this occurs. Also, we suggest disabling
+preventive GCs which cause OOME when the heap is almost full.
```

Comment:
Note that this paragraph doesn't explain every single flag that exists above.
It would be too detailed. Let's not add it here.

Reply:
agreed.

Reply:
How about adding after this paragraph?

We enable -XX:+UnlockDiagnosticVMOptions and -XX:+UseAESCTRIntrinsics to improve AES performance for S3, etc. on ARM64 ([JDK-8271567](https://bugs.openjdk.java.net/browse/JDK-8271567))

something like this

`-XX:-G1UsePreventiveGC` is disabled along with `-XX:+UseAESCTRIntrinsics` option for performance issue (`JDK-8293861 <https://bugs.openjdk.org/browse/JDK-8293861>`)